P2_Ward_Stake_Area.sql

```diff
@@ -0,0 +1,49 @@
+
+					--AREA BY AREA_UNIT_NUMBER
+
+ WITH area AS (
+				SELECT DISTINCT 
+ 				  parent_unit_number as AREA_UNIT_NUMBER, 
+ 				  CASE
+ 					WHEN PARENT_UNIT_NAME LIKE '%Salt%'                 THEN 'Utah SLC'
```

Comment:
I know this makes it easier to know where final query is, but lets just indent one time here like we saw today in the code standard

Reply:
The indents in github make it look like it is way out there... It looks a lot cleaner in ssms or azure. I will try to change everything the exact way that you want it though.